docs/guides/transactions/running-background-jobs.md

```diff
@@ -178,48 +178,59 @@ Handle.GET("/Hello", () =>
 });
 ```
 
-## Await with scheduled tasks
+## Task scheduling and async/await
 
-Since .NET pick the thread when using the `await` keyword, there's no way to ensure that the code after an awaited scheduled task is executed on a Starcounter thread or .NET thread.
+Async/await programming in Starcounter is the same as in standard .NET. The only difference is that if there is a Starcounter-related code (SQL, transactions, etc.) that comes after the `await` statement - such code should be wrapped into a delegate given to `Scheduling.RunTask`. Below is an example demonstrating this. It uses delayed `Response`. Note also the use of `Db.TransactAsync` - its needed so that Starcounter scheduler is not creating additional threads.
```

Comment:
its -> it's

that Starcounter scheduler -> that the Starcounter scheduler

scheduler is not creating -> scheduler does not create